client/account-recovery/reset-password/reset-password-form/reset-option-set.jsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+
+const getResetOptionDisplayString = ( optionName, translate ) => {
+	switch ( optionName ) {
+		case 'primary':
+			return translate( 'main', {
+				comment: 'It is about which way a user wants to reset the password. e.g. main email, recovery email.',
```

Comment:
Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'Account Recovery' )` *ES Score: 12.04*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/account-recovery/reset-password-form)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).